user_guide_src/source/database/query_builder.rst

```diff
@@ -1084,11 +1084,13 @@ Class Reference
 		Compiles and runs SELECT statement based on the already
 		called Query Builder methods.
 
-	.. php:method:: getWhere([$where = NULL[, $limit = NULL[, $offset = NULL]]])
+	.. php:method:: getWhere([$where = NULL[, $limit = NULL[, $offset = NULL[, $returnSQL = FALSE[, $reset = TRUE]]]]])
 
 		:param	string	$where: The WHERE clause
 		:param	int	$limit: The LIMIT clause
 		:param	int	$offset: The OFFSET clause
+		:param 	bool $returnSQL: If true, returns the generate SQL, otherwise executes the query.
+	    :param 	bool $reset: Are we want to clear query builder values?
```

Comment:
Thanks! Small items:
* Indent looks to be 1 tab short
* Should read "**Do** we want to clear..."

Reply:
Done.